composer/Dockerfile

```diff
@@ -1,11 +1,11 @@
-# syntax=docker/dockerfile:1.2.1
+# syntax=docker/dockerfile:1.4.3
 ARG repository=local
 ARG tag=latest
-ARG alpine=3.15.0
-FROM --platform=$BUILDPLATFORM ${repository}/download:${tag} AS download 
+ARG alpine=3.16.2
```

Comment:
Instead of specifying the patch version would it make sense to just target the minor version?

Reply:
In this case, no. As that could produce different images depending on the time in which it is executed, even for the same commit to this repository. Ideally, a given commit in `isle-buildkit` should always produce the same image if built locally, remotely, now or in three months in the future. Our goal being stable reproducible builds.